packages/flutter_tools/test/integration.shard/ios_content_validation_test.dart

```diff
@@ -243,6 +243,7 @@ void main() {
         <String>['file', simulatorAppFrameworkBinary.path],
       );
       expect(archs.stdout, contains('Mach-O 64-bit dynamically linked shared library x86_64'));
+      expect(archs.stdout, contains('Mach-O 64-bit dynamically linked shared library arm64'));
```

Comment:
This isn't actually related to this change, but this case should have been added with #85642